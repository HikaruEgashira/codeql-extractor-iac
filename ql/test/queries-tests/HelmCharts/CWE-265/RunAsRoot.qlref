security/HelmCharts/CWE-265/RunAsRoot.ql