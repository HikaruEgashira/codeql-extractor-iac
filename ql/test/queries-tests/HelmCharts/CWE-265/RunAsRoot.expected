| Chart.yml:9:5:11:2 | SecurityContext | root user set |
| Chart.yml:31:9:32:25 | SecurityContext | root user set |
